Description: update desktop icon chinese Name and Comment and add mimetype
 .
 This patch is come from atzlinux project.
 https://www.atzlinux.com
 .
 The orig git commit info is:
 .
 https://gitee.com/atzlinux/gmchess/blob/0e9f9d01c17cd356132ebab17bc4998faa204da2/debian/patches/0004-fix-desktop-mime.patch
 .
Author: careone <emacslocale@126.com>

---
Last-Update: 2022-02-13

diff --git a/data/gmchess.desktop.in b/data/gmchess.desktop.in
index 56fc1b6..05cc528 100644
--- a/data/gmchess.desktop.in
+++ b/data/gmchess.desktop.in
@@ -1,12 +1,13 @@
 [Desktop Entry]
 Name=Chinese chess
-Name[zh_CN]=天书棋谈
+Name[zh_CN]=天书象棋
 Comment=Play the popular chinese chess game
-Comment[zh_CN]=玩流行的中国象棋游戏
-Exec=gmchess
+Comment[zh_CN]=中国象棋游戏，支持人机对战、打谱、保存棋谱
+Exec=gmchess %f
 Icon=gmchess
 Terminal=false
 Type=Application
+MimeType=application/xiangqi-pgn;application/xiangqi-xqf;application/xiangqi-mxq;application/xiangqi-che;application/xiangqi-ccm;application/xiangqi-chn;application/xiangqi-xqi;
 Keywords=Xiangqi;Chess;Chinese;
 Keywords[zh_CN]=Xiangqi;Chess;Chinese;象棋;中国象棋;
 Categories=GTK;Game;

